euth_wagtail/assets/scss/components/forms/_general-form.scss

```diff
@@ -78,6 +78,23 @@
             content: "\f05A "; //info-circle
         }
     }
+
+    header {
+        &.form-inner {
+            padding-top: 6rem;
```

Comment:
I know Opin in inconsistent but for paddings and margins we would like to always use the $spacer variable so if we ever change the size, all spaceings are adjusted, not just some.